app/templates/briefs/_brief_response_data.html

```diff
@@ -27,17 +27,19 @@
             {% elif item.type == "dynamic_list" %}
               {% for question in item.questions %}
                   {% if question.type == "boolean" %}
-                    {% if question._data['followup'] %}
+                    {% if question._data['values_followup'] %}
```

Comment:
I spent quite a lot of changes going over this and changing it to values_followup is what made it work, actually. We can look it over on Monday if you want.